README.md

```diff
@@ -1,23 +1,184 @@
-# @pagopa/io-react-native-integrity
+# io-react-native-integrity
 
-React Native package which bridges Andoird Play Integrity API and iOS App Attest
+React Native interfaces for managing secure storage in iOS and Android.
 
 ## Installation
 
 ```sh
-npm install @pagopa/io-react-native-integrity
+npm install io-react-native-integrity
```

Comment:
Trying to install `io-react-native-integrity` returns an error
`npm ERR! 404 Not Found - GET https://registry.yarnpkg.com/io-react-native-integrity - Not found
`

Reply:
Works well with `@pagopa/io-react-native-integrity`

Reply:
Fixed in fa3ecb5df3419bd1001f1e20952ef71d3c12ba2a, thanks.